### PR TITLE
Added compatibility for latest testing suite.

### PR DESCRIPTION
**Pull request by leonhelmus**
* Added ext-simplexml as a requirement.
* Fixed unit tests to make it compatible with phpunit ^9.0

### CODE REVIEW

File: tests/FilesystemTest.php [hunk 1]
```diff
@@ -67,12 +67,11 @@ public function testRead(
     /**
      * @return void
      *
-     * @expectedException \RuntimeException
-     *
      * @covers ::read
      */
     public function testReadExceptionReadable()
     {
+        $this->expectException(\RuntimeException::class);
```
Comment (ksangers) [inline tests/FilesystemTest.php:74]: Nitpicky here. Please `use` the class. Same for others below

File: tests/XmlAccessorTest.php [hunk 2]
```diff
@@ -111,12 +111,11 @@ public function testGetDescendant(
      *
      * @dataProvider getDescendantDataProvider
      *
-     * @expectedException \InvalidArgumentException
-     *
      * @covers ::getDescendant
      */
     public function testGetDescendantException()
     {
+        $this->expectException(\InvalidArgumentException::class);
```
Comment (ksangers) [inline tests/XmlAccessorTest.php:118]: As earlier: `use`

**Review by ksangers — CHANGES_REQUESTED** [review]

**Review by ashokadewit — APPROVED** [review]

**Review by ksangers — APPROVED** [review]

**Comment by ksangers** [comment]
@leonhelmus can you get this to run through scrutinizer as well?

**Comment by leonhelmus** [comment]
Fixed it and merging it now.

**Pull request merged**